### PR TITLE
Issue #372: Add xccdf:TestResult@test-system attribute

### PR DESCRIPTION
**Pull request by jan-cerny**
The xccdf:TestResult element should contain a test-system
attribute which is defined in specification as follows:
Name of the benchmark consumer program that generated this
<xccdf:TestResult> element; SHOULD be either a CPE name or a
CPE applicability language expression.

### CODE REVIEW

**Comment by jan-cerny** [comment]
This pull request fixes issue https://github.com/OpenSCAP/openscap/issues/372

**Comment by ybznek** [comment]
I don't see problem. Could you provide some simple test, please?

**Comment by jan-cerny** [comment]
@ybznek Yes, I could.

File: tests/API/XCCDF/unittests/test_xccdf_test_system.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+name=$(basename $0 .sh)
+result=$(mktemp -t ${name}.out.XXXXXX)
+stderr=$(mktemp -t ${name}.out.XXXXXX)
```
Comment (ybznek) [inline tests/API/XCCDF/unittests/test_xccdf_test_system.sh]: IMHO, `.err.XXXXXX)`  is better for stderr than `.out.XXXXXX`

Comment (jan-cerny) [inline tests/API/XCCDF/unittests/test_xccdf_test_system.sh]: Nice! Thank you.

**Comment by jan-cerny** [comment]
I have added a very simple test for this.

**Pull request merged**